openwisp_ipam/tests/test_admin.py

```diff
@@ -426,3 +426,31 @@ def assert_response(response):
                 reverse('admin:ipam_export_subnet', args=[subnet.id]), follow=True
             )
             assert_response(response)
+
+    def test_subnet_32(self):
```

Comment:
Why are you adding these tests in `test_admin.py`, when the code for these tests is in `api.py`? These tests should be in `test_api.py`, please pay close attention to these details, we need to maintain the codebase clean and readable, I almost did not notice this.

Reply:
My bad :worried: , moved tests to `test_api.py` :+1: